minecraft_servers/zyneonstudios/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+    "server_name": "zyneonstudios",
+    "nice_name": "Zyneon Studios",
+    "direct_ip": "zyneonstudios.com",
+    "social": {
+      "web": "https://www.zyneonstudios.com",
+      "twitter": "zyneonstudios",
+      "youtube": "https://www.youtube.com/nerotvlive",
+      "discord": "https://discord.com/invite/DYKArt7A7D"
+    },
+    "discord": {
+      "server_id": 892032464384585768,
+      "rename_to_minecraft_name": true

```

Comment:
Please use `false`, as this is the default value. We will accept `true` as a value here only for verified partners.

Reply:
done